etl/storage/storage.go

```diff
@@ -93,14 +91,10 @@ func (s *Storage) SavePulse(pulse models.Pulse) error {
 	timer := prometheus.NewTimer(SavePulseDuration)
 	defer timer.ObserveDuration()
 
-	s.savePulseLock.Lock()
-	defer s.savePulseLock.Unlock()
-	timerInside := prometheus.NewTimer(SavePulseExecutionDuration)
```

Comment:
We probably should remove `SavePulseExecutionDuration` from `etl/storage/storage_metrics.go` too

Reply:
sure, maybe later, if it helps ?

Reply:
later is fine)